lesson14/style.css

```diff
@@ -0,0 +1,35 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+  .button {
+    @apply bg-teal-600 hover:bg-teal-400 text-white font-semibold hover:text-white py-2 px-4 border border-teal-500 hover:border-transparent rounded top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 fixed;
+  }
+  .modal-button {
+    @apply bg-teal-600 hover:bg-teal-400 text-white font-semibold py-2 px-4 border-teal-500 hover:border-transparent absolute inset-x-0 bottom-0 h-1/6;
+  }
+  .overlay {
+    @apply hidden fixed top-0 left-0 w-full h-full bg-slate-500 opacity-50;
+  }
+
+  .modal {
+    @apply hidden fixed top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 lg:w-1/3 lg:h-1/2 md:w-1/2 md:h-1/2 w-3/4 h-1/2 bg-white border border-slate-300 rounded;
+  }
+
+  .close-button {
+    @apply absolute top-2 right-5 cursor-pointer;
+  }
+  .modal-title {
+    @apply lg:text-2xl md:text-xl text-base text-center  lg:py-28 md:py-20 py-10;
+  }
+  .flexbox {
+    @apply flex flex-col items-center  fixed top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2;
+  }
+  .input-field {
+    @apply block h-10 border border-slate-500 rounded fixed top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2;
+  }
+  .modal-validation {
+    @apply block hidden text-rose-500 text-center pt-20;
+  }
+}
```

Comment:
nits
In TailwindCSS, you should not use @apply just to make your HTML look pretty. 
If you make everything a component with @apply, you'll have to think of a class name every time.
We recommend stopping at buttons and small components.
One reason is that it defeats the advantage of TailwindCSS, which is that you don't have to think about class names every time.

Using @apply is difficult, but here's the official documentation.
https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply